Wikipedia/Code/WelcomeAnimationExtensions.swift

```diff
@@ -0,0 +1,753 @@
+import Foundation
```

Comment:
This file is really big - and contains some distinct extensions / objects - so we should break this up